src/blocks/sound.rs

```diff
@@ -37,6 +37,19 @@
 //! device_kind = "source"
 //! ```
 //!
+//! Display warning in block if microphone if using the wrong port:
+//!
+//! ```toml
+//! [[block]]
+//! block = "sound"
+//! driver = "pulseaudio"
+//! device_kind = "source"
+//! format = " $icon { $volume|}$active_port "
+//! [block.mappings]
+//! "alsa_input.pci-0000_00_1f.3.analog-stereo analog-input-rear-mic" = ""
+//! "alsa_input.pci-0000_00_1f.3.analog-stereo analog-input-front-mic" = "ERR!"
```

Comment:
I'm a bit confused. Is your `active_port` "analog-input-rear-mic" or the entire string?

Reply:
The former

Reply:
Then how does it work?

Reply:
Oh, it doesn't.